src/v0/staff.ts

```diff
@@ -36,17 +36,17 @@ export class Staff {
         let uri = `${this.options.base}${this.endpoint}/${this.options.user}`
         let isQuery = false
         if (query) {
-          if (query.start !== null) {
+          if (query.start) {
```

Comment:
Can't you use the qs library to make the querystring? Or were you concerned about 'null' being put into a querystring? It looks like above you changed the StaffQuery to be 'undefined' and in that case the qs library would completely omit that property anyways.